src/tools/clippy/clippy_utils/src/usage.rs

```diff
@@ -189,7 +194,7 @@ impl<'tcx> Visitor<'tcx> for ReturnBreakContinueMacroVisitor {
                 } else {
                     rustc_hir::intravisit::walk_expr(self, ex);
                 }
-            },
+            }
```

Comment:
Can you please revert all those formatting changes?

Reply:
I didn't notice I am reverting it.